go.sum

```diff
@@ -8,6 +8,7 @@ github.com/apparentlymart/go-textseg v1.0.0 h1:rRmlIsPEEhUTIKQb7T++Nz/A5Q6C9IuX2
 github.com/apparentlymart/go-textseg v1.0.0/go.mod h1:z96Txxhf3xSFMPmb5X/1W05FF/Nj9VFpLOpjS5yuumk=
 github.com/armon/go-radix v1.0.0 h1:F4z6KzEeeQIMeLFa97iZU6vupzoecKdU5TX24SNppXI=
 github.com/armon/go-radix v1.0.0/go.mod h1:ufUuZ+zHj4x4TnLV4JWEpy2hxWSpsRywHrMgIH9cCH8=
+github.com/aws/aws-sdk-go v1.16.36/go.mod h1:KmX6BPdI08NWTb3/sm4ZGu5ShLoqVDhKgpiN924inxo=
 github.com/aws/aws-sdk-go v1.17.0 h1:+pbWEdKxH1qlLb07as1+auEVvx+IxkaDzQLwMzbK1tI=
 github.com/aws/aws-sdk-go v1.17.0/go.mod h1:KmX6BPdI08NWTb3/sm4ZGu5ShLoqVDhKgpiN924inxo=
```

Comment:
this looks like you didn't run `tidy`, not sure, maybe redo and re vendor?

Reply:
I did previously, sorry for not including it in the original PR comment. 🙁 Re-running it now shows no changes.

```sh
$ git checkout td-aws-sdk-go-base
Branch 'td-aws-sdk-go-base' set up to track remote branch 'td-aws-sdk-go-base' from 'origin'.
Switched to a new branch 'td-aws-sdk-go-base'
$ go mod tidy
$ go mod vendor
$ git status
On branch td-aws-sdk-go-base
Your branch is up to date with 'origin/td-aws-sdk-go-base'.

nothing to commit, working tree clean
```